.github/workflows/Test.yml

```diff
@@ -39,10 +40,23 @@ jobs:
         run: |
           python -m venv venv
 
-      - name: Install tzfpy and test
+      - name: Install tzfpy and test(not windows)
+        if: matrix.os != 'windows-latest'
         run: |
           source venv/bin/activate
           pip install --upgrade pip
           pip install -r requirements_dev.txt
           maturin develop
           pytest --cov
+
+      - name: Install tzfpy and test(windows)
+        shell: bash
+        # for windows need maturin build
+        if: matrix.os == 'windows-latest'
+        run: |
+          . venv/Scripts/activate.bat
```

Comment:
venv is not required if use `maturin build`

Reply:
just found that....